keyboards/planck/keymaps/skug/keymap.c

```diff
@@ -15,158 +15,127 @@
  */
 
 #include QMK_KEYBOARD_H
+#include "keymap_swedish.h"
 #include "muse.h"
 
 extern keymap_config_t keymap_config;
 
+#define _______ KC_TRNS
```

Comment:
Lines 23-24 should be deleted. QMK defines these automatically for every keymap.

https://github.com/qmk/qmk_firmware/blob/27e216fc1a481bb3e65832458bd2b199cb499e0b/quantum/quantum_keycodes.h#L29-L31

Reply:
Oh so it seems, resolved in https://github.com/qmk/qmk_firmware/pull/4133/commits/3120557e988e5527ab3fc7e25b52537cf2c01d54